src/main/kotlin/no/nav/familie/oppdrag/repository/OppdragLager.kt

```diff
@@ -1,41 +1,44 @@
 package no.nav.familie.oppdrag.repository
 
-import no.nav.familie.kontrakter.felles.objectMapper
 import no.nav.familie.kontrakter.felles.oppdrag.OppdragId
 import no.nav.familie.kontrakter.felles.oppdrag.OppdragStatus
 import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
 import no.nav.familie.kontrakter.felles.oppdrag.behandlingsIdForFørsteUtbetalingsperiode
 import no.nav.familie.oppdrag.iverksetting.Jaxb
 import no.nav.familie.oppdrag.iverksetting.OppdragMapper
+import no.trygdeetaten.skjema.oppdrag.Mmel
 import no.trygdeetaten.skjema.oppdrag.Oppdrag
+import org.springframework.data.annotation.Id
 import org.springframework.data.relational.core.mapping.Column
 import java.time.LocalDateTime
+import java.util.*
 
-data class OppdragLager(val fagsystem: String,
+data class OppdragLager(@Id
```

Comment:
@Id må vel settes etter att man har lagt inn id'er på alle?
Tror dette må skje i ennå ett til steg

Reply:
Vi må ha @Id for å få lov til å lagre med crudRepository.